wasmplugin/plugin.go

```diff
@@ -365,3 +431,35 @@ func logError(error ctypes.MatchedRule) {
 		proxywasm.LogDebug(msg)
 	}
 }
+
+type AddressInfo struct {
+	IP   string
+	port int
+}
+
+// Parsing address (e.g. "127.0.0.1:8080") to retrieve the IP
+func parseAddress(rawAddress []byte) (string, error) {
+	// Split address and port
+	address := string(rawAddress)
```

Comment:
Maybe better to work with bytes and convert to string at the end?

Reply:
Thank you JC, done